README.md

```diff
@@ -1,84 +1,145 @@
 # 3id-blockchain-utils
+
 [![standard-readme compliant](https://img.shields.io/badge/readme%20style-standard-brightgreen.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)
 [![CircleCI](https://img.shields.io/circleci/project/github/ceramicnetwork/js-3id-blockchain-utils.svg?style=for-the-badge)](https://circleci.com/gh/ceramicnetwork/js-3id-blockchain-utils)
 [![npm](https://img.shields.io/npm/dt/3id-blockchain-utils.svg?style=for-the-badge)](https://www.npmjs.com/package/3id-blockchain-utils)
 [![npm](https://img.shields.io/npm/v/3id-blockchain-utils.svg?style=for-the-badge)](https://www.npmjs.com/package/3id-blockchain-utils)
 [![Codecov](https://img.shields.io/codecov/c/github/ceramicnetwork/js-3id-blockchain-utils.svg?style=for-the-badge)](https://codecov.io/gh/ceramicnetwork/js-3id-blockchain-utils)
 
-This package contains a bunch of utilities that is used by 3ID and 3Box in order to create and verify links from blockchain addresses.
+Utility functions for linking blockchain accounts to DID.
 
 ## Tabel of Contents
+
+- [Overview](#overview)
+  - [Linking](#linking)
+  - [Validation](#validation)
 - [Install](#install)
 - [Usage](#usage)
 - [Supported blockchains](#supported-blockchains)
-- [Contributing](#contributing)
-  - [Test](#Test)
-- [License](#license)
+- [Adding support for a blockchain](#adding-support-for-a-blockchain)
+- [Test](#test)
+- [Maintainers](#maintainers)
+- [Licence](#licence)
+
+## Overview
+
+To fully link blockchain account one has to:
+
+- prove ownership of the account via [Linking](#linking)
+- [validate](#validation) the link.
+
+Due to asymmetry in dependencies - linking is light weigth, while validation requires quite heavy code - and target use cases,
+the package is divided into two: namely `@3id-blockchain-utils/linking` and `@3id-blockchain-utils/validation`.
```

Comment:
Is this the namespace we want to use rather than `@ceramicnetwork`?

Reply:
We should use `@ceramicnetwork/blockchain-linking` and `@ceramicnetwork/blockchain-validation`